spec/spec_helper.rb

```diff
@@ -11,32 +11,34 @@
   config.color = true
 end
 
-def stub_get(path, fixture_name, params={})
```

Comment:
I realized this was testing Faraday specifics so I removed it. My HMAC stuff uses RestClient and the OAuth stuff uses Faraday which have subtle differences in headers. I think it's better to just test the headers this gem adds.